python/perspective/README.md

```diff
@@ -2,42 +2,50 @@
 
 Python APIs for [perspective](https://github.com/finos/perspective) front end
 
-[![npm](https://img.shields.io/npm/v/@finos/perspective.svg?style=flat-square)](https://www.npmjs.com/package/@finos/perspective)
```

Comment:
I realize that this is out-of-date, but it is worth asking _why_ it is out-of-date.  Implied rules for project maintenance are not scalable to a project of this size.

If this is to be a duplicate of the project readme, can we add this copy to `.gitignore` and duplicate it as part of the python build please - or replace it altogether with a trivial readme that just links to the project `README.md` on GitHub?

Reply:
I'm good with that, it has been a slightly different variant of the main README for a while now, not sure why it passed review but deleting and copying on build is better

Reply:
fixed